$NetBSD: patch-ab,v 1.1 2005/03/16 12:02:20 rillig Exp $

Let's be polite and let the system first include its headers before
Python redefines everything.

--- twisted/runner/portmap.c.orig	Thu Dec  5 08:59:31 2002
+++ twisted/runner/portmap.c	Wed Mar 16 03:41:57 2005
@@ -19,9 +19,9 @@
 
 /* portmap.c: A simple Python wrapper for pmap_set(3) and pmap_unset(3) */
 
-#include <Python.h>
 #include <rpc/rpc.h>
 #include <rpc/pmap_clnt.h>
+#include <Python.h>
 
 static PyObject * portmap_set(PyObject *self, PyObject *args)
 {
